pytest_logger/plugin.py

```diff
@@ -13,6 +14,28 @@
 else:
     string_type = str
 
+os_symlink = getattr(os, "symlink", None)
+if callable(os_symlink) is False and platform.system() == 'Windows':
+    
+    def symlink_win(source, linkname):
+        """ Symlink implementation on Windows platform, working from Vista
+        inspired by http://stackoverflow.com/questions/6260149/os-symlink-support-in-windows
+        """
+        import ctypes
+        flags = 1 if source is not None and os.path.isdir(source) else 0
+        try:
+            linkname = linkname.replace('/', '\\')
+            csl = ctypes.windll.kernel32.CreateSymbolicLinkW
```

Comment:
I'm actually wondering what are all the possibilities of doing win32/python2 symlinks:
  - is this the only API one could use or
  - is there some command one could use from prompt (os.system/subprocess) or
  - how was this problem solved in python3 implementation?